docs/src/notebooks/3_basic_stdmodel_usage.jl

```diff
@@ -174,10 +174,16 @@ pgm_duplicate
 #
 check_duplicate_reaction(pgm_duplicate, model.reactions; only_metabolites = false) # can also just check if only the metabolites are the same but different stoichiometry is used
 
-# ## Checking the internals of `StandardModel`s: `is_mass_balanced`
+# ## Checking the internals of `StandardModel`s: `reaction_is_mass_balanced`
 
-# Finally, [`is_mass_balanced`](@ref) can be used to check if a reaction is mass
+# Finally, [`reaction_is_mass_balanced`](@ref) can be used to check if a reaction is mass
```

Comment:
`reaction_mass_balanced` is the way to name it. :D

Reply:
will change :)